tests/sparseml/pytorch/sparsification/pruning/test_mfac.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import torch
+
+from sparseml.pytorch.sparsification.pruning import (
+    FisherInverseFast,
+    FisherInverseFastBlock,
+    FisherInverseFastSmallBlocks,
+)
+
+
+# precent-wise precision in terms of the mean of the resulting tensors
+PRECISION = 0.000001
+
+
+@pytest.mark.skipif(
+    os.getenv("NM_ML_SKIP_PYTORCH_TESTS", False),
+    reason="Skipping pytorch tests",
+)
+def test_fast_small_blocks():
```

Comment:
Duplicating Ben's comment, let's pack it all inside `pytest.mark.parameterize`

Reply:
(`small_blocks`, `large_blocks` etc this could all be possibly one function with different args from `pytest.mark.parameterize`)